bin/plot_krun_results

```diff
@@ -211,10 +229,13 @@ def add_inset_to_axis(axis, rect):
     return fig.add_axes([fig_left, fig_bottom, fig_width, fig_height])
 
 
-def draw_subplot(axis, data, cycles_data, title, x_range, y_range, window_size, outliers,
-                 unique, common, median, tukey):
+def draw_subplot(axis, data, cycles_data, aperf_data, mperf_data, title,
```

Comment:
I think if we find some time, we should pack all of these data points into some data structure, so we can pass fewer arguments, but now now ;)